CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Update input value when I try to update with an option selected in the `AutocompleteInput` component
+
+### Fixed
```

Comment:
```suggestion
```

Reply:
@jardelymaris can you apply this suggests?